clients/gtest/CMakeLists.txt

```diff
@@ -44,6 +44,7 @@ endif( )
 
 set(rocblas_test_source
     rocblas_gtest_main.cpp
+    ${Tensile_TEST_SRC}
```

Comment:
Tensile_TEST_SRC is moved to the front of the linker list, in order that the GEMM tests occur first during Google Tests runs. This allows quicker debugging of GEMM tests without having to specify fancy Google Test filters. It immediately runs GEMM tests, and will fail with error if anything is wrong with them.